modules/imgcodecs/src/ios_conversions.mm

```diff
@@ -116,6 +118,8 @@ void UIImageToMat(const UIImage* image,
         if (!alphaExist)
             bitmapInfo = kCGImageAlphaNoneSkipLast |
                                 kCGBitmapByteOrderDefault;
+        else
+            m = cv::Scalar(0);
```

Comment:
yes, that's what I was saying about misaligned data access making setting only the alpha potentially slower than just setting all 4 channels.  In my case, the cost is not something necessary and reduces performance, hence my intentionally leaving that out when the image isn't using an alpha channel.   For real time video frame processing you really don't want any unnecessary computation cost.   I could add a comment in the header file about this if you think that's useful.